base/server/src/main/java/org/dogtagpki/server/rest/v2/ACLFilter.java

```diff
@@ -51,14 +76,37 @@ public abstract class ACLFilter extends HttpFilter {
     private static final String LOGGING_MISSING_ACL_MAPPING = "ACL mapping not found; OK";
     private static final String LOGGING_INVALID_ACL_MAPPING = "internal error: invalid ACL mapping";
     private Properties aclProperties;
-    private String acl;
+    private String defaultAcl;
+    private Map<String, String> aclMap;
 
     @Override
     public void doFilter(ServletRequest request, ServletResponse response, FilterChain chain) throws IOException, ServletException {
-
+        String method;
+        String path;
+        String acl = defaultAcl;
         if(request instanceof HttpServletRequest req &&
                 response instanceof HttpServletResponse resp) {
+            method = req.getMethod();
+            path = req.getPathInfo() != null ? req.getPathInfo() : "";
+            final String aclSearch = method + ":" + path;
+            if (aclMap!=null) {
+                String[] aclKeys = aclMap.keySet().stream().filter( key -> {
+                    String keyRegex = key.replaceFirst("\\*", ".*").replace("\\{\\}", "([A-Za-z0-9_\\-]*)");
+                    return aclSearch.matches(keyRegex);
+                } ).toArray(String[]::new);
```

Comment:
Not an expert on streams, is it possible to get the map values directly? Something like:
```
String[] acls = aclMap.entrySet().stream()
      .filter(e -> { ... })
      .map(Map.Entry::getValue)
      .toArray(String[]::new);

if (acls.length > 0) {
    acl = acls[0];
}
```

Reply:
I have modified the stream sorting the keys in reverse order. In this case, the "*" will always be last and it is possible to collect the first element. In this way the loop among the element is not needed.